cvprac/cvp_api.py

```diff
@@ -56,7 +58,53 @@
 }
 
 
-class CvpApi(object):
+def sanitize_warnings(data):
+    ''' Sanitize the warnings returned after validation.
```

Comment:
a few typos in the docstring, correct version:

Sanitize the warnings returned after validation.

In some cases where the configlet has both errors
and warnings, CVP may split warnings that have
`,` across multiple strings.
This method concatenates the strings back into one string
per warning, and corrects the warningCount.

Args:
data (dict): A dict that contains the result
of the validation operation
Returns:
response (dict): A dict that contains the result of the
validation operation